setup.py

```diff
@@ -28,7 +28,7 @@
     # update with a tag from the nawi-hub/idaes-pse
     # when a version of IDAES newer than the latest stable release from PyPI
     # will become needed for the watertap development
-    "idaes-pse==2.5.0",
+    "idaes-pse @ git+https://github.com/IDAES/idaes-pse@dynamic_cv0d",
```

Comment:
Is this branch supposed to exist? The only thing I could find is IDAES/idaes-pse#1460, which has already been merged, so using `main` should be enough:

```suggestion
    "idaes-pse @ git+https://github.com/IDAES/idaes-pse@main",
```

More in general, if you want to use the _name_ of a PR's head branch, you have to use the corresponding fork rather than `idaes-pse`. Otherwise, you can use the PR number by using the "special" ref GitHub creates.

In other words, the following should be equivalent (assuming for the sake of example that IDAES/idaes#1460 is still open and the branch still exists on your fork):

```
# option 1: using the fork and the branch name
idaes-pse @ git+https://github.com/adam-a-a/idaes-pse@dynamic_cv0d

# option 2: using the upstream repo and the PR number
idaes-pse @ git+https://github.com/IDAES/idaes-pse@refs/pull/1460/merge
```